src/main/io/ledstrip.c

```diff
@@ -568,27 +567,27 @@ static void applyLedBatteryLayer(bool updateNow, uint32_t *timer)
     static bool flash = false;
 
     int state;
-    int timeOffset = 1;
+    int frequency = 1;
```

Comment:
It would be slightly better to use `timerDeltaUs`(with better name) and assign constant expression into it according to selected case